packages/devops/scripts/lambda/helpers/creation.py

```diff
@@ -157,3 +167,43 @@ def create_instance(
         setup_subdomains_via_gateway(deployment_type, new_instance_object, subdomains_via_gateway, deployment_name)
 
     return new_instance_object
+
+def create_db_instance_from_snapshot(
+  instance_name,
+  snapshot_name,
+  instance_type,
+  security_group_code=None,
+  security_group_id=None,
+):
+    security_group_ids = get_security_group_ids_config(security_group_code, security_group_id)
+
+    snapshot_id = get_latest_db_snapshot(snapshot_name)
+    print('Starting to clone db instance from snapshot')
+    rds.restore_db_instance_from_db_snapshot(
+        DBInstanceIdentifier=instance_name,
+        DBSnapshotIdentifier=snapshot_id,
+        DBInstanceClass=instance_type,
+        Port=5432,
+        PubliclyAccessible=True,
+        VpcSecurityGroupIds=security_group_ids,
+        Tags=[
+            {
+                'Key': 'DeploymentName',
+                'Value': instance_name
+            },
+        ],
```

Comment:
Not actually sure if this is necessary, since the db identifier is just the deployment name anyway... but felt consistent with the way we handle tagging of ec2 instances 👍

Reply:
Yep and can we add the deployment type tag for consistency too?